plugin.xml

```diff
@@ -30,7 +30,7 @@
   <js-module src="www/com.salesforce.util.push.js" name="util.push"></js-module>
 
   <engines>
-    <engine name="cordova-android" version="6.2.3" />
+    <engine name="cordova-android" version="6.4.0" />
     <engine name="cordova-ios" version="4.4.0" />
   </engines>
```

Comment:
The plugins have new versions too. See https://cordova.apache.org/news/2017/11/10/plugins-release.html
Should we move to those as well?

Reply:
Sure - let me do that in a separate PR after I upgrade `iOS` as well.

Reply:
Ok